econml/validate/sensitivity_analysis.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) PyWhy contributors. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from econml.utilities import _safe_norm_ppf
+
+def sensitivity_interval(theta, sigma, nu, cov, alpha, c_y, c_t, rho, interval_type='ci'):
+    """Calculate the sensitivity interval."""
+    if interval_type not in ['theta', 'ci']:
```

Comment:
Should we create a type that exposes both of these and nicer pretty-printing, more like what we return from effect_inference (as opposed to effect_interval)?

Reply:
I had a similar thought. Maybe a `sensitivity_summary` method which also exposes sensitivity intervals + robustness values.